### PR TITLE
DOC: Remove makePanel from docs (#25609)

### LINKED ISSUE #25609: Remove makePanel Calls in Docs
As a follow up to #25231 some exceptions are getting raised when building the docs as a result of calls to makePanel lingering around. Here's at least one section where this is occurring:

http://pandas-docs.github.io/pandas-docs-travis/getting_started/dsintro.html#deprecate-panel

Would want to remove these references and possibly the entire section surrounding to clean up doc build

### PR DESCRIPTION
**Pull request by ArtificialQualia**
- [x] closes #25609
- [ ] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

As described in #25609, I removed the tm.makePanel() calls from the docs, along with some of the surrounding sections (a few "Deprecate Panel" sections).  I did not replace these calls with an equivalent manual creation of a Panel.

If desired, I could create a panel manually in the docs with something like this:

```
cols = ['Item' + c for c in string.ascii_uppercase[:K - 1]]	
data = {c: tm.makeTimeDataFrame() for c in cols}	
p = Panel.fromDict(data)
```

but this PR is currently as requested in the issue with the sections removed.

### CODE REVIEW

File: doc/source/whatsnew/v0.13.1.rst [hunk 1]
```diff
@@ -222,60 +222,6 @@ Enhancements
 
 - Panel :meth:`~pandas.Panel.apply` will work on non-ufuncs. See :ref:`the docs<basics.apply>`.
 
-  .. ipython:: python
-
-     import pandas.util.testing as tm
```
Comment (WillAyd) [inline doc/source/whatsnew/v0.13.1.rst]: Hmm I don't think we want to change old whatsnew items like this but I'm not sure how we've handled removals like this historically either - @TomAugspurger maybe you can offer insight?

Comment (jreback) [reply doc/source/whatsnew/v0.13.1.rst]: yeah don't change these at all, rather convert them to code-blocks

**Review by jreback — CHANGES_REQUESTED** [review]

**Comment by ArtificialQualia** [comment]
I've converted all the old whatsnew entries to code-blocks.  I did not do that for the 'Deprecate Panel' section in dsintro, if you would like me to do it there as well just let me know.

**Review by WillAyd — CHANGES_REQUESTED** [review]
General approach lgtm though you have some lint failures in CI - could you take a look?

**Comment by ArtificialQualia** [comment]
I've fixed the linting errors.  I did have to add a `# noqa E999` in `v0.13.1.rst` as flake8-rst was unable to parse that code-block.

**Review by WillAyd — APPROVED** [review]
lgtm @jreback

**Review by jreback — APPROVED** [review]

**Comment by jreback** [comment]
thanks @ArtificialQualia

**Pull request merged**